src/components/device/ace.ts

```diff
@@ -266,7 +266,32 @@ export default class Ace implements IDeviceManager, IUVCControls {
   }
 
   getState(): Promise<any> {
-    throw new Error('Method not implemented.');
+    return new Promise(async (resolve, reject) => {
+      try {
+        const cnnFeature = new huddly.CnnFeature();
+        cnnFeature.setFeature(huddly.Feature.AUTOZOOM);
+        const azStatus = await this.getCnnFeatureStatus(cnnFeature);
+
+        if (azStatus.hasAzStatus()) {
+          resolve({
+            autozoom_enabled: azStatus.getAzStatus().getAzEnabled(),
+          });
+        }
+      } catch (err) {
+        reject(err);
+      }
```

Comment:
Maybe do a `promise().then().catch()` style instead of `try...catch`? Just nitpicking